target/classes/META-INF/resources/matchingPage.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

```

Comment:
Is this matchingPage file also in the webapp folder? Have you tried deleting the extra repeated files in META-INF and seeing if the code still works? 

I wonder if something strange happened with the merge to cause this.

Reply:
Yes, the matchingPage file is also in the webapp folder. I tried to delete them but everytime I run the command "mvn package exec:java" appeared again.

I'll try to uncommit those changes and ignore the files.